ui/src/components/view/DetailsTab.vue

```diff
@@ -51,6 +51,21 @@
               {{ dataResource.rootdisksize }} GB
             </div>
           </div>
+          <div v-else-if="['template', 'iso'].includes($route.meta.name) && item === 'size'">
+            <div>
+              {{ parseFloat(dataResource.size / (1024.0 * 1024.0 * 1024.0)).toFixed(2) }} GB
```

Comment:
something for future maybe, we can have method for converting size into string

Reply:
Yeah I couldn't figure how to have like a global utility of some kind (because then to use it, you'll need to import it everywhere). @shwstppr suggestions welcome.